src/platform/tls_openssl.c

```diff
@@ -1462,6 +1462,30 @@ CxPlatTlsSecConfigCreate(
         }
     }
 
+    if (CredConfigFlags & QUIC_CREDENTIAL_FLAG_USE_CACERTFILE) {
+        if (CredConfig->CaCertificateFile) {
+            Ret = SSL_CTX_load_verify_locations(SecurityConfig->SSLCtx,
```

Comment:
Would you mind updating the formatting to match more like the call to `SSL_CTX_use_certificate` above?

Reply:
Sorry, I've been working in another repo with wierd indentation lately :-)